packages/terra-button/src/_themes.scss

```diff
@@ -0,0 +1,84 @@
+@import './mixins';
+
+@include cerner-consumer-theme {
+  --terra-consumer-color: #64696c;
+  --terra-consumer-color-dark-30: #383f42;
+  --terra-consumer-color-dark-50: #1c1f21;
+  --terra-consumer-color-light-30: rgba(#383f42, 0.3);
+  --terra-consumer-color-light-50: rgba(#64696c, 0.5);
+  --terra-consumer-color-light-65: rgba(#383f42, 0.65);
+  @include terra-consumer-theme-button();
+
+  /* stylelint-disable selector-class-pattern */
+  /* stylelint-disable suitcss/custom-property-no-outside-root */
+
+  .terra-consumer-theme-concept1 {
+    --terra-consumer-color: #c24726;
+    --terra-consumer-color-dark-30: #97371e;
+    --terra-consumer-color-dark-50: #6c2715;
+    --terra-consumer-color-light-30: rgba(#c24726, 0.3);
+    --terra-consumer-color-light-50: rgba(#c24726, 0.5);
+    --terra-consumer-color-light-65: rgba(#c24726, 0.65);
+    @include terra-consumer-theme-button();
```

Comment:
actually the variables defined in `cerner-consumer-theme` can't access those defined in the nested classes. CSS custom property behaviors like js closure. In the example:
```
.cerner-consumer-theme {
     --terra-button-background-color-primary: var(--terra-consumer-color);
    .terra-consumer-theme-concept1 {
         --terra-consumer-color: #123456;
    }
}

<div class=cerner-consumer-theme>
   <div class=terra-consumer-theme-concept1>
       <button>
   </div>
</div>
```
in the outside div, `--terra-button-background-color-primary: var(--terra-consumer-color);` is evaluated to `invalid value` since `--terra-consumer-color` is not defined in its ancestors. in the inner div, there are two variables available due to cascading:
```
--terra-button-background-color-primary: invalid variable;
--terra-consumer-color: #123456;
```